src/main/java/xbot/common/properties/TableProxy.java

```diff
@@ -6,15 +6,20 @@
 
 import java.util.Hashtable;
 
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
 /**
  * A simple implementation of ITableProxy. Uses a HashTable as the table to store and save properties.
  * 
  * @author Alex
  */
+@Singleton
```

Comment:
IIRC this one was intentionally not a singleton because we have multiple separate TableProxy instances in certain scenarios.

Reply:
I think it was related to unit tests where we want there to be separate TableProxy instances for the in-memory vs smart-dashboard objects.

Reply:
Yes my mistake, this shouldn't be singleton